etc/kayobe/environments/ci-aio/kolla/globals.yml

```diff
@@ -11,3 +11,8 @@ openstack_service_rpc_workers: "1"
 
 docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
 docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}/gpg"
+
+# Elasticsearch memory tuning
+es_heap_size: 1g
+
+rabbitmq_monitoring_user: monitoring
```

Comment:
Will this be inherited from the base setting? I.e we could remove it?

Also, I think it is a relic left over from the RabbitMQ exporter container. Now we use the RabbitMQ plugin, I think we can remove it entirely?

Reply:
Correct and correct; I copied this from original Will's patch